core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTestBase.scala

```diff
@@ -150,7 +150,7 @@ trait SplitClusterDUnitTestBase extends Logging {
     doTestRowTableCreation()
   }
 
-  def _testComplexTypesForColumnTables_SNAP643(): Unit = {
+  def testComplexTypesForColumnTables_SNAP643(): Unit = {
```

Comment:
@ashetkar Why do you anticipate that this test will fail in security test? Can you add some minimal method to base class to override and set credentials in SplitClusterDUnitSecurityTest so that all the tests get run there too with security (including the three which have been overridden as no-op)? I think it will be useful to run all the tests with security to confirm that those scenarios work (unless you think those are covered elsewhere already). In that need to handle just this complex type handling in security dunit because AFAIK there is no separate coverage for complex types in security tests.